CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ These are the section headers that we use:
 
 ### Added
 
+- Argilla server docker image is built with PostgreSQL support. Closes [#2686](https://github.com/argilla-io/argilla/issues/2686)
```

Comment:
Perhaps this should be under "Changed".

So this won't be included in 1.7.0, right?